requirements.txt

```diff
@@ -2,4 +2,4 @@ psycopg2
 pydantic
 python-systemd
 requests
-rohmu >= 1.0.7
+git+https://github.com/Aiven-Open/rohmu.git@97fc675b8993ed9a29d5d86d27edd891318b4448#egg=rohmu
```

Comment:
Technically, we could merge while pinning this. But it's probably not the way we want to do things.